ablation_eval.py

```diff
@@ -254,6 +306,7 @@ def create_job_configs(
                     "model": model_kwargs,
                     "tokenizer_name": main_config.tokenizer_name,
                     "scheduler": main_config.scheduler,
+                    "optimizer": task_config.get("optimizer", main_config["optimizer"]),
```

Comment:
Given that none of the existing eval configs have an optimizer defined, we should add the ability to fall back to the task optimizer defaults:
```suggestion
                    "optimizer": task_config.get("optimizer", main_config.get("optimizer", None)),
```